pkg/controller/grafanadashboard/dashboard_controller.go

```diff
@@ -94,7 +94,9 @@ func (r *ReconcileGrafanaDashboard) Reconcile(request reconcile.Request) (reconc
 			// Return and don't requeue
 			return reconcile.Result{}, nil
 		}
-		// Error reading the object - requeue the request.
+		// Error reading the object - requeue the r
```

Comment:
Seems like formatting messed up this comment.

Reply:
good spot, updated